From ab50299efa6e8dcd29b45aab41f8242ab17ef35a Mon Sep 17 00:00:00 2001
From: romi <r@sansfontieres.com>
Date: Fri, 22 Dec 2023 04:51:21 +0000
Subject: [PATCH 144/188] Add .glif to XML file-types (#9130)

`.glif` files are standard files in the type design industry. From the
Unified Font Object specification website:

The Glyph Interchange Format (GLIF) is a simple and clear XML
representation of a single glyph. GLIF files typically have a .glif
extension.
https://unifiedfontobject.org/versions/ufo3/glyphs/glif/
---
 languages.toml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/languages.toml
+++ b/languages.toml
@@ -2320,7 +2320,8 @@ file-types = [
   "xpdl",
   "xul",
   "xoml",
-  "musicxml"
+  "musicxml",
+  "glif"
 ]
 indent = { tab-width = 2, unit = "  " }
 
